Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'foreman-tasks', '>= 0.7.20'
```

Comment:
That would mean plugins currently using foreman-tasks would not need to specify them in the depedant gems anymore, right?